packages/atomic/cypress/e2e/generated-answer-assertions.ts

```diff
@@ -4,16 +4,21 @@ import {should} from './common-assertions';
 import {GeneratedAnswerSelectors} from './generated-answer-selectors';
 
 export function assertLogOpenGeneratedAnswerSource(log: boolean) {
-  it(`${should(log)} log a openGeneratedAnswerSource click event`, () => {
-    if (log) {
-      cy.expectCustomEvent('generatedAnswer', 'openGeneratedAnswerSource');
-    } else {
-      cy.wait(50);
-      cy.wrap(AnalyticsTracker)
-        .invoke('getLastCustomEvent')
-        .should('not.exist');
-    }
-  });
+  if (log) {
+    cy.expectCustomEvent('generatedAnswer', 'openGeneratedAnswerSource');
+  } else {
+    cy.wait(50);
+    cy.wrap(AnalyticsTracker).invoke('getLastCustomEvent').should('not.exist');
```

Comment:
nit, ideally we would not wait an arbitrary amount of time then assert something is not there.
We would wait for something that is certain to be there/occur because of the Act part of the test, and then Assert that the thing that should not be there is not there.